lib/ansible/modules/user.py

```diff
@@ -7,7 +7,7 @@
 
 
 DOCUMENTATION = r"""
-module: user
+module: freebsduser
```

Comment:
`user` is a generic module which works on different OSes. Changing to `freebsduser` is not correct.